docs/source/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'OpenSearch Python Client'
+copyright = '2022, OpenSearch Project Contributors'
```

Comment:
```suggestion
copyright = 'OpenSearch Project Contributors'
```

Reply:
Thanks for contribution. IIRC, OpenSearch decided not to include year in copyright headers/footers.

Reply:
Thanks - I changed it.